### PR TITLE
Control panel errors/failures

### PR DESCRIPTION
**Pull request by kshitijSharma2014**

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez 
please review these.

**Comment by puneet0191** [comment]
@kshitijSharma2014  Tested this one, can you fix this issue:

image

I would suggest to remove that string from Assertion, it is not required

File: tests/system/webdriver/Pages/System/ControlPanelPage.php [hunk 1]
```diff
@@ -39,7 +39,6 @@ class ControlPanelPage extends AdminPage
 			'Template Manager' => 'administrator/index.php?option=com_templates',
 			'Language Manager' => 'administrator/index.php?option=com_languages',
 			'Install Extensions' => 'administrator/index.php?option=com_installer',
-			'All extensions are up-to-date' => 'administrator/index.php?option=com_installer&view=update',
```
Comment (kshitijSharma2014) [inline tests/system/webdriver/Pages/System/ControlPanelPage.php:42]: @puneet0191  you asked me to remove this right or something else?

Comment (puneet0191) [inline tests/system/webdriver/Pages/System/ControlPanelPage.php:42]: Yeap

**Comment by puneet0191** [comment]
@kshitijSharma2014 Looks good...tested PR, Merging it now Thanks :+1:

**Pull request merged**